.gitignore

```diff
@@ -23,3 +23,8 @@
 *.so
 *.o
 *.a
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
**/__pycache__

Reply:
Do you have any reason for the suggestion?
The patch is from github reference.

Reply:
yes, `__pycache__` ignores pycache at the root folder only.
`**/__pycache__` ignores pycache folder in the whole repository.